.github/actions/spelling/allow/apis.txt

```diff
@@ -35,6 +35,7 @@ dataobject
 dcomp
 debugbreak
 delayimp
+DECRQRCA
```

Comment:
Instead of adding the incorrect spelling to the dictionary, could we not just use the correct spelling in the code? 😉 It should be `DECRQCRA`.